package com.example.demo.dto.request;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;

public class ForgotPasswordRequest {

    @NotBlank(message = "email không được bỏ trống")
    @Email(message = "email không đúng định dạng")
    private String email;

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }
}
